contrib/agent/build.gradle

```diff
@@ -1,11 +1,17 @@
 plugins {
   id 'com.github.johnrengelman.shadow' version '2.0.0'
+  id 'me.tatarka.retrolambda' version '3.6.0'
 }
 
 description = 'OpenCensus Agent'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+sourceCompatibility = JavaVersion.VERSION_1_7
+targetCompatibility = JavaVersion.VERSION_1_7
+
+// Experimental support for legacy Java 1.6 JVMs.
```

Comment:
IIUIC, the root project takes care of this (animalsniffer) already. Signature for the agent is 
signature 'org.codehaus.mojo.signature:java16:+@signature'